puppet/zulip/files/nginx/zulip-include-frontend/app

```diff
@@ -83,3 +83,10 @@ location /api/ {
 
 include /etc/nginx/zulip-include/uploads.route;
 include /etc/nginx/zulip-include/app.d/*.conf;
+
+# Serve the public-only realm export.
+location /public {
+    include /etc/nginx/zulip-include/api_headers;
+
+    alias /home/zulip/uploads;
+}
```

Comment:
I don't think it makes sense to add a custom path here.  For one thing, this configuration is shared between the S3 and local file uploads backend.  But also, I think it'd be cleaner to just use the existing directory structure (I don't think `public` in the URL gives the right idea).  I'll post a specific proposal below.